### PR TITLE
Pro Tools Theme for Audacium

### PR DESCRIPTION
**Pull request by ClausMartini**
*(short description of the changes and the motivation to make the changes)*
An optional theme for those familiar with the Avid DAW.

- [x] I made sure the code compiles on my machine.
- [x] I made sure there are no unnecessary changes in the code.
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving.
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?".
- [x] I hereby reaffirm that I am licensing my contribution under the GNU General Public License v2.0 or later (`GPL-2.0-or-later`).

### CODE REVIEW

File: src/Theme.cpp [hunk 1]
```diff
@@ -1281,10 +1292,15 @@ auStaticText::auStaticText(wxWindow* parent, wxString textIn) :
...
-void auStaticText::OnPaint(wxPaintEvent & WXUNUSED(evt))
-{
+    }
+    <<<<<<< HEAD
+    
+    void auStaticText::OnPaint(wxPaintEvent & WXUNUSED(evt))
+    =======
+    
+    void auStaticText::OnPaint(wxPaintEvent & evt)
+    >>>>>>> 5d38ecf9a... Added Pro Tools theme
```
Comment (AnErrupTion) [inline src/Theme.cpp:1302]: Can you explain those "comments"? They don't look like comments to me and they look like they will make errors.

**Review by AnErrupTion — CHANGES_REQUESTED** [review]
You seem to have forgot to put the "protools" theme name in function `teThemeType ThemeBase::ThemeTypeOfTypeName( const wxString & Name )` in Theme.cpp.

File: src/Theme.cpp (hunk 1 above)
Comment (ClausMartini) [reply src/Theme.cpp:1302]: Thanks for pointing that out. And you're right. Those comments is just part of something I've been working on. Forgot to remove it. I'll push an update.

File: src/Theme.cpp [hunk 2]
```diff
@@ -198,6 +198,11 @@ static const unsigned char HiContrastImageCacheAsData[] = {
 #include "HiContrastThemeAsCeeCode.h"
 };
 
+
+static const unsigned char protools_template_iconImageCacheAsData[] = {
+#include "ProToolsThemeAsCeeCode.h"
```
Comment (AnErrupTion) [inline src/Theme.cpp:203]: You have to rename the variable to ProToolsImageCacheAsData because that's what you specified in code.

**Review by AnErrupTion — CHANGES_REQUESTED** [review]

**Comment by AnErrupTion** [comment]
No, not the file name, the variable name (which is currently `protools_template_iconImageCacheAsData`. Revert your change and rename the variable to `ProToolsImageCacheAsData`.

File: src/Theme.cpp (hunk 2 above)
Comment (ClausMartini) [reply src/Theme.cpp:203]: Should work fine now. Sorry about that.

**Comment by ClausMartini** [comment]
If you wish I can make more themes for audacium.

**Pull request merged**